content-src/components/Base/_Base.scss

```diff
@@ -5,6 +5,14 @@
   min-height: 100vh;
   padding: ($section-spacing + $section-vertical-padding) $base-gutter $base-gutter;
 
+  &.ds-outer-wrapper-breakpoint-override {
```

Comment:
I see another instance of this class further down the file, is it intended to have it twice?

Reply:
Yeah, they're for different selectors.